sdk/tests/deqp/data/gles3/shaders/linkage.test

```diff
@@ -495,8 +495,12 @@ group varying "Varying linkage"
 
 		# different interpolation
 		case differing_interpolation_2
+			# This deviates from C++ dEQP here. GLSL ES 3.00.4 specifies interpolation
+			# qualifier of variables with the same name declared in all linked shaders
+			# must match, in Section 4.3.9 Interpolation.
 			version 300 es
 			desc "varying interpolation different (smooth vs. centroid)"
+			expect link_fail
```

Comment:
Thanks for pointing that. 
I just see your patch in angle should fix this bug: https://chromium-review.googlesource.com/#/c/359607/.  Close this pr.